src/Seksjon/Personalia.elm

```diff
@@ -120,7 +121,10 @@ nesteSamtaleSteg model tekst samtaleSeksjon =
     Model
         { model
             | aktivSamtale = samtaleSeksjon
-            , historikk = model.historikk ++ [ Bruker tekst ]
+            , seksjonsMeldingsLogg =
+                model.seksjonsMeldingsLogg
+                    |> MeldingsLogg.leggTilSpørsmål (samtaleTilMeldingsLogg samtaleSeksjon)
+                    |> MeldingsLogg.leggTilSvar (Melding.svar [ tekst ])
```

Comment:
Kanskje disse egentlig er i feil rekkefølge? For spørsmålene fra _neste_ samtaleseksjon burde jo legges til etter svaret fra _forrige_ seksjon? Med mindre jeg ikke helt skjønner hva som skjer 🤔